QuickfireQuestions.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css/style.css">
```

Comment:
you're using bootstrap but your style sheet is loaded before bootstrap
i.e. you have linked it in before bootstrap css, it should come after.

Reply:
Hi Anthony, I have swapped them around now, so they should be the right way round.

Thank you